source/common/http/header_map_impl.cc

```diff
@@ -279,7 +278,7 @@ bool HeaderMapImpl::operator==(const HeaderMapImpl& rhs) const {
 }
 
 void HeaderMapImpl::insertByKey(HeaderString&& key, HeaderString&& value) {
-  StaticLookupEntry::EntryCb cb = static_lookup_table_.find(key.c_str());
+  StaticLookupEntry::EntryCb cb = ConstSingleton<StaticLookupTable>::get().find(key.c_str());
```

Comment:
Question: With this pattern, would there be a copy of the static in each translation unit? I suspect there will be and in general it seems like not what we want, but I'm not sure. (Unrelated to this change really but just thought about it).